api/routes.go

```diff
@@ -2,60 +2,48 @@ package api
 
 import (
 	"encoding/json"
-	"fmt"
 	"math"
 	"net/http"
 	"sort"
 	"strconv"
 	"time"
-
 	// MySQL driver
-	"gopkg.in/cas.v1"
-
 	log "github.com/Sirupsen/logrus"
 	"github.com/gorilla/mux"
-
 	"github.com/wtg/shuttletracker/model"
+	"gopkg.in/cas.v1"
 )
 
 //RouteIsActive determines if the current time means a route should be active or not
 //TODO: Move this to updater
 func (api *API) RouteIsActive(r *model.Route) bool {
```

Comment:
I'm not sure about this method. Why does it need to modify the route in the database? Can the API just determine if each route is active after it pulls them from the database?

I think we shouldn't try to store/persist data (whether a route is active) if it can be reasonably calculated from other data (route activation times). Gives us one source of truth, is easier to reason about, and prevents some weird issues related to caching and locking.

Additionally, this method appears to do two things: modify route activation status for the API, and return a boolean for testing. It should just do one thing, and the tests can verify that the result is as expected through another manner.